### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by Goleay** [billkarsh/Alignment_Projects#4]
When I check mon-*.err after I run msub.sht, I found Segmentation fault (core dumped). And my test data is 25 16-bit tif pictures which are 2644 x 2048.

**Comment by billkarsh** [comment]
Hi Goleay, Wow, honored that you are trying the aligner! Happy to see if we can get you going. Let's take this conversation offline. Email me here: karshb@hhmi.org. My first question is if you downloaded and ran the Alignment Tutorials project. That should run perfectly. If it does, then we know all of the executables are properly compiled and there is something problematic in your data. In that case I will want to get sample images and scripts from you to diagnose your specific input issue. Let me know if sounds good.

**Comment by billkarsh** [comment]
BTW, Today I reran the alignment tutorial myself...I hadn't used the aligner or even the Janelia cluster for many years. The IT group had gone through several generations of cluster OS and math libraries over that time, so I had to rebuild everything to get the aligner executables up to date. But I ran all successfully and today's results agree with the documented results in the Tutorial. So the code and methods remain sound.

**Comment by billkarsh** [comment]
Spoke too soon! I received EM images from Goleay and aligned them without incident with resin masking turned off. However, with resin masking enabled there was indeed a seg. fault during montaging (lsqw). I traced this to a few CPOINT2 entries having an illegal rgn index. Although dmesh has a procedure (Tabulate) to screen out points that map between illegal regions, it was coded wrong: rgn == 0 should always be flagged as illegal. Fixed, and thank you very much Goleay!
